### ISSUE TITLE
GoA Colour palette

### ISSUE THREAD

**Issue by twjeffery** [GovAlta/ui-components#144]
# What is being submitted?
We need to extend the GoA colour palette to allow for greater flexibility within our design system.

**Comment by twjeffery** [comment]
I have added a proposed reorganization/naming based on how each colour is used, as well as extended status colours (dark, mid, light for each status.) 

https://www.figma.com/file/ylmHeuDMfxnDBnP1VaQYz8/DIO-DS---Guidelines?node-id=1308%3A3748

**Comment by twjeffery** [comment]
@Spark450 @Saul-png 
It would be great to review this together and discuss how we want to move things forward.

**Comment by Spark450** [comment]
Will discuss "new" status colors shown [here](https://xd.adobe.com/view/e0d4486a-480d-4609-9ecb-3628c075796d-b587/screen/b3d3a6ff-1213-403b-98f4-31e4c4164149/specs/?hints=off) with CPE

**Comment by Spark450** [comment]
- After discussion with CPE on changes to status colors it was determined that these changes are not fully approved at this time and that the DIO should move forward with the existing set of status colours in the color palette.

- As per discussion with CPE current palette **can be extended** to suit the needs of the DIO services.

**Comment by twjeffery** [comment]
### Next Steps

- [ ] Need to resync with @chrisolsen on specific colours to add
- [ ] next steps on organization/naming, and uses
- [ ] communicate that this is open for new ideas, submissions, updates to the colours

**Comment by tarynlaramie** [comment]
for reference: https://medium.com/eightshapes-llc/tokens-in-design-systems-25dd82d58421

**Comment by Spark450** [comment]
As per discussion with @tarynlaramie @twjeffery @chrisolsen We have decided on the following naming convention and mapping.
[Color mapping.csv](https://github.com/GovAlta/ui-components/files/7830674/Color.mapping.csv)

**Comment by twjeffery** [comment]
@chrisolsen  As I worked the colours into the Figma design library, I had to slightly change 3 of the colour names so that they showed up within the design tool appropriately. I wanted to run the change by you as well so that we keep the naming across design and development as close as possible:

- --goa-color-text changed to --goa-color-text-primary

- --goa-color-interactive changed to --goa-color-interactive-default

- --goa-color-brand changed to --goa-color-brand-default

With the original naming, it created a sort of double label for the “default” colour within the sections that we have defined. Adding “primary” or “default” fixes that on our end, but let me know what you think.

Eg.
![image](https://user-images.githubusercontent.com/82968683/149042344-84712a22-3f10-44d2-9311-efcecd2f0729.png)
![image](https://user-images.githubusercontent.com/82968683/149042372-bfd514f6-410b-4c83-b33b-426ba3ea3454.png)
